.golangci.yml

```diff
@@ -7,5 +7,6 @@ linters:
     - govet
     - ineffassign
     - staticcheck
+    - tparallel
```

Comment:
How does `tparallel` know when it's appropriate to run a test in parallel? There seem to be [a number of open issues](https://github.com/moricho/tparallel/issues) that demonstrate this linter isn't foolproof.

Reply:
Agree. Remove it